### ISSUE TITLE
Spanish to English Translation of code for easier interpretability

### ISSUE THREAD

**Issue by samtholathrobin** [ruxailab/web-eye-tracker#16]
Within the web-eye-tracker/app/services repository there are 3 files that are dash.py, data_visualization.py and data_exploration.py that have the analytical elements written in Portuguese.  I propose the translation and addition of these text in English (either as comments within code or by rewriting source code) for the easy interpretability of other contributors. 

I believe that this is a quality of life improvement that will help many contributors in the future. 
Please do let me know what you think of this issue and I would be happy to help.

**Comment by KarinePistili** [comment]
Yes you can translate them, but be careful to not mislead any translation.

I believe something that would also be helpful would be to make some sort of documentation within the functions of the project to help on code understanding. Maybe some comments or something similar.

Would you like to work in this as well?

**Comment by sitamgithub-MSIT** [comment]
> Yes you can translate them, but be careful to not mislead any translation.
> 
> I believe something that would also be helpful would be to make some sort of documentation within the functions of the project to help on code understanding. Maybe some comments or something similar.
> 
> Would you like to work in this as well?

Yes. If anybody is not currently taking this up, then I would like to work on it!

**Comment by samtholathrobin** [comment]
> Yes you can translate them, but be careful to not mislead any translation.
> 
> I believe something that would also be helpful would be to make some sort of documentation within the functions of the project to help on code understanding. Maybe some comments or something similar.
> 
> Would you like to work in this as well?

Sure I'd love to work on it. 
I will keep your points in mind and work accordingly.

**Comment by KarinePistili** [comment]
I'll be assigning it to @samtholathrobin as it is the person who raised the issue and asked to work on it first.

But thanks for the interest @sitamgithub-MSIT !

**Comment by samtholathrobin** [comment]
@KarinePistili Is there any way I can get the dataset associated with these files? 
The dataset contains values associated with the left and right irises. Are these generated from the web app?
